### PR TITLE
feat(@vtmn/svelte): optimization of `VtmnModal` component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Changes description

- Optimisation of slots
- Title is now a slot
- Slots don't include parent class
- Renaming of props to be consistent
- Removal of the `show` class because it creates bugs

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- Yes, Change the title prop into a slot

### CODE REVIEW

File: packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte [hunk 1]
```diff
@@ -31,11 +31,8 @@
       show = false;
     }}
   >
-    <p
-      slot="description"
-      id="vtmn-modal-description"
-      class="vtmn-modal_content_body--text"
-    >
+    <slot slot="title">This is the title</slot>
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte:34]:
```
<svelte:fragment slot="title">
This is the title
</svelte:fragment>
````

File: packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte [hunk 2]
```diff
@@ -31,11 +31,8 @@
...
+    <slot slot="description">
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte:35]:
```
<svelte:fragment slot="description">
This is the title
</svelte:fragment>
```

File: packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte [hunk 3]
```diff
@@ -67,8 +64,8 @@
       dolores molestias exercitationem accusamus perferendis est aut repudiandae
       optio vel dicta reprehenderit ad, repellendus officiis cumque omnis labore
       in quia?
-    </p>
-    <div slot="actions" class="vtmn-modal_content_actions">
+    </slot>
+    <slot slot="actions">
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte:68]:
```
<svelte:fragment slot="actions">
[...]
</svelte:fragment>
```

File: packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte [hunk 4]
```diff
@@ -102,11 +99,8 @@
       show = false;
     }}
   >
-    <div
-      slot="description"
-      id="vtmn-modal-description"
-      class="vtmn-modal_content_body--text"
-    >
+    <slot slot="title">This is the title</slot>
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte:102]:
```
<svelte:fragment slot="title">
[...]
</svelte:fragment>
```

File: packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte [hunk 5]
```diff
@@ -102,11 +99,8 @@
...
+    <slot slot="description">
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnModal/VtmnModal.stories.svelte:103]:
```
<svelte:fragment slot="description">
[...]
</svelte:fragment>
```

File: packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js [hunk 6]
```diff
@@ -103,6 +94,14 @@ describe('VtmnModal', () => {
       await expectedCancelOnElement(getCloseButton(container), component, 1);
     });
 
+    test('Should have a slot title', () => {
+      const { container } = render(VtmnModal, {
+        show: true,
+        title: 'Unit-test',
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js:100]: No more props `title`

File: packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js [hunk 7]
```diff
@@ -120,6 +119,13 @@ describe('VtmnModal', () => {
     });
 
     describe('With slot description', () => {
+      test('Should have a slot title', () => {
+        const { container } = render(VtmnModalWithDescription, {
+          show: true,
+          title: 'Unit-test',
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js:125]: No more props `title`

File: packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModalWithSlots.svelte [hunk 8]
```diff
@@ -3,6 +3,7 @@
 </script>
 
 <VtmnModal {...$$restProps}>
+  <p slot="title">title</p>
   <p slot="description">description</p>
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModalWithSlots.svelte:7]: use `<svelte:fragment>`

File: packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModalWithDescription.svelte [hunk 9]
```diff
@@ -3,5 +3,6 @@
 </script>
 
 <VtmnModal {...$$restProps}>
+  <p slot="title">title</p>
   <p slot="description">description</p>
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModalWithDescription.svelte:7]: use `<svelte:fragment>`

File: packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js [hunk 10]
```diff
@@ -103,6 +94,14 @@ describe('VtmnModal', () => {
       await expectedCancelOnElement(getCloseButton(container), component, 1);
     });
 
+    test('Should have a slot title', () => {
+      const { container } = render(VtmnModal, {
+        show: true,
+        title: 'Unit-test',
+      });
+      expect(getSlotTitle(container)).toBeVisible();
+    });
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnModal/test/VtmnModal.spec.js:103]: Check if slot have class `vtmn-modal_content_title--text`

**Review by GaspardMathon — COMMENTED** [review]
Thanks @Tlahey  :)

**Review by lauthieb — CHANGES_REQUESTED** [review]
This causes problem with existing modal in React.
Let's check this together in videoconference tomorrow. :)

**Review by lauthieb — APPROVED** [review]

**Review by Tlahey — APPROVED** [review]

**Review by lauthieb — APPROVED** [review]

**Pull request merged**